google-cloud-pubsub/OVERVIEW.md

```diff
@@ -174,16 +174,32 @@ pubsub = Google::Cloud::PubSub.new
 
 sub = pubsub.subscription "my-topic-sub"
 
-subscriber = sub.listen do |received_message|
+# Create a subscriber to listen for available messages.
+# By default, this block will be called on 8 concurrent threadsk
+# but this can be tuned with the :threads option.
+# The :streams and :inventory parameters allow further tuning.
```

Comment:
Use markdown code backticks: 

```
`streams` and `inventory`
```

No need for `:` because these aren't symbols.